tools/autopasTools/generators/RandomGenerator.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <array>
+#include <cmath>
```

Comment:
Why was this done?

Reply:
For some reason that is totally not understandable to me, this file suddenly wouldn't compile anymore because some things were undefined. Even though I didn't touch this

Reply:
Reason: `RandomGenerator` always needed the `cmath` include, however previously, it was included transitively from somewhere else. Probably Markus' PR removed the source of the transitive relation. So adding this here is the right thing to do.